db/migrate/20180220091854_create_trips.rb

```diff
@@ -0,0 +1,17 @@
+class CreateTrips < ActiveRecord::Migration[5.1]
+  def change
+    create_table :trips do |t|
+      t.bigint "group_id"
```

Comment:
A question: why are you using `bigint` here?

Reply:
Probably from using references when creating the foreign key

Reply:
Yes, it come up automatically with the rails generator.